eva_submission/eload_backlog.py

```diff
@@ -58,20 +58,19 @@ def get_species_info(self):
         self.eload_cfg.set('submission', 'scientific_name', value=sci_name)
 
         with get_metadata_conn() as conn:
-            query = f"select distinct b.vcf_reference_accession " \
+            query = f"select distinct b.analysis_accession, b.vcf_reference_accession " \
                     f"from project_analysis a " \
                     f"join analysis b on a.analysis_accession=b.analysis_accession " \
                     f"where a.project_accession='{self.project_accession}' and b.hidden_in_eva=0;"
             rows = get_all_results_for_query(conn, query)
         if len(rows) < 1:
             raise ValueError(f'No reference accession for {self.project_accession} found in metadata DB.')
-        elif len(rows) > 1:
-            raise ValueError(f'Multiple reference accession for {self.project_accession} found in metadata DB.')
-        asm_accession, = rows[0]
-        self.eload_cfg.set('submission', 'assembly_accession', value=asm_accession)
-        fasta_path, report_path = get_reference_fasta_and_report(sci_name, asm_accession)
-        self.eload_cfg.set('submission', 'assembly_fasta', value=fasta_path)
-        self.eload_cfg.set('submission', 'assembly_report', value=report_path)
+        for analysis_accession, asm_accession in rows:
+            print(self.eload_cfg.query('submission', 'analyses', analysis_accession))
```

Comment:
Is this still needed?  If so maybe should be `self.info` or `debug`.

Reply:
Forgot to remove it